pepys_import/core/store/postgres_db.py

```diff
@@ -138,9 +138,9 @@ class Platform(BasePostGIS):
 
     platform_id = Column(UUID(as_uuid=True), primary_key=True, default=uuid4)
     name = Column(String(150), nullable=False)
-    pennant = Column(String(150), nullable=False)
-    trigraph = Column(String(3), nullable=False)
-    quadgraph = Column(String(4), nullable=False)
+    pennant = Column(String(150))
```

Comment:
@BarisSari - we only need 10 chars for the pennant, thanks.

Reply:
Char size of pennant changed:
https://github.com/debrief/pepys-import/pull/51/commits/84f3e5c3ca0a25f74bddba5d0af8d36a7a7015a2